pkg/apis/pipeline/v1beta1/pipelinerun_types.go

```diff
@@ -527,9 +526,9 @@ type PipelineRunTaskRunStatus struct {
 type PipelineRunRunStatus struct {
 	// PipelineTaskName is the name of the PipelineTask.
 	PipelineTaskName string `json:"pipelineTaskName,omitempty"`
-	// Status is the RunStatus for the corresponding Run
+	// Status is the CustomRunStatus for the corresponding Run
```

Comment:
```suggestion
	// Status is the CustomRunStatus for the corresponding CustomRun
```

Reply:
Actually, these should all be "CustomRun or Run", since `PipelineRunRunStatus` now applies to both. Fixing!